conformance/1.0/kfp-conformance.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
```

Comment:
Do we need to add a Copyright section for these files?

Reply:
Done.